### DISCUSSION TITLE
How can I use v-ripple directive in jsx?

### DISCUSSION THREAD

**Discussion by zhongjiaguo001** [quasarframework/quasar#14375] [General - Components / Directives / etc]
### Description

Using the v-ripple directive in jsx, my control reported an error

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Directives (quasar)

**Comment by pdanpdan** [comment]
Please provide details (fill the issue template and provide the code you are using)

**Comment by zhongjiaguo001** [comment]
![image](https://user-images.githubusercontent.com/44691489/189831077-9dcdf0ce-c00d-4636-8d9d-32d44c9c30fe.png)
![image](https://user-images.githubusercontent.com/44691489/189831098-a8f26fce-0ce1-4139-b56f-bb0c824144d1.png)
How can I properly use v-ripple directive in jsx？

**Comment by pdanpdan** [comment]
You can check here: https://github.com/vuejs/jsx-vue2/issues/141
And you should try searching google for `vue3 vite use directives in jsx`
